requirements.txt

```diff
@@ -1,5 +1,8 @@
-tensorflow==1.15.2
 tqdm==4.45.0
 matplotlib==3.2.1
 numpy==1.18.3
 opencv-python==4.2.0.34
+tensorflow~=2.2.0rc3
```

Comment:
tensorflow>=1.15.*

Reply:
not use tensorflow-gpu==1.15?

Reply:
Right, it should be tensorflow-gpu>=1.15.*. Have you tested on Tensorflow 2?

Reply:
Nope. Did you do it on Windows 10 when developing?

Reply:
Oh, I thought you added these requirements because you have tested. I tested it on Ubuntu 18.04 with Tensorflow>=1.15.0. I'll change these requirements to my environment.

Reply:
ok